src/index.html

```diff
@@ -1,13 +1,47 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <div class="card" data-qa="card">
+        <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac image"
+        >
+        <h2 class="card__description">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <p class="card__number">Код товара: 195434</p>
+
+        <div class="card__review">
+          <div class="stars-review stars-review--4">
+            <div class="stars-review__star"></div>
+            <div class="stars-review__star"></div>
+            <div class="stars-review__star"></div>
+            <div class="stars-review__star"></div>
+            <div class="stars-review__star"></div>
+          </div>
+
+          <p class="card__stars-review">Отзывов: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-tag">Цена:</p>
+          <h3 class="card__price-value">69 999 грн</h3>
```

Comment:
```suggestion
          <p class="card__price-value">69 999 грн</p>
```
It's unnecessary to use here H3, better change on <p> or <span>